compose/release-notes.md

```diff
@@ -7,6 +7,16 @@ redirect_from:
   - /release-notes/docker-compose/
 ---
 
+## 1.25.5
+(2020-04-10)
+
+### Features
+
+- Bump OpenSSL from 1.1.1d to 1.1.1f
+
```

Comment:
Could you remove the extra line?

Reply:
I don't see any extra line. Line 16 is just to separate the 2 features

Reply:
Yea, an extra line isn't required between bulleted list items. We can ignore this for now as most of the entries within the Compose release notes have an extra line to separate list items.